release/ray_release/command_runner/anyscale_job_runner.py

```diff
@@ -67,6 +67,9 @@ def __init__(
         self.prepare_commands = []
         self._wait_for_nodes_timeout = 0
 
+        self._results_uploaded = True
+        self._metrics_uploaded = True
+
```

Comment:
what is this

Reply:
Fail fast path - if we know we have not uploaded metrics or results from the cluster to s3, we don't need to retry 3 times when trying to download them

Reply:
can you please add a comment?